<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acc_sh</class>
 <widget class="QDialog" name="acc_sh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>风作用顶点最大加速度计算</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>164</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_us">
      <property name="text">
       <string>体型系数u_s</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_hft_3">
      <property name="text">
       <string>基本风压w0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_w">
      <property name="text">
       <string>10年重现期风压w(kn/m^2)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_xi">
      <property name="text">
       <string>脉动增大系数xi</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_nu">
      <property name="text">
       <string>脉动影响系数nu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_h">
      <property name="text">
       <string>建筑物高度H</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_b">
      <property name="text">
       <string>建筑物迎风宽度B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_l">
      <property name="text">
       <string>建筑物顺风向长度L</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_m">
      <property name="text">
       <string>建筑物总质量(t)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_ccd">
      <property name="text">
       <string>地面粗糙度类别</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_hfznb">
      <property name="text">
       <string>横风向一阶结构阻尼比zata_s1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_hft">
      <property name="text">
       <string>横风向第一自振周期T_1(s)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_hft_2">
      <property name="text">
       <string>横风向一阶振型指数beta</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>81</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="le_us"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_w0"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_w"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_xi"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_nu"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_h"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_b"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_l"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_m"/>
    </item>
    <item>
     <widget class="QComboBox" name="cb_ccd">
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_hfznb"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_hft"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_beta"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="le_sf">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>顺风向顶点最大加速度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnyes">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>490</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_hf">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>横风向顶点最大加速度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_hf">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>计算过程见shacc.txt</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pte_h">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>51</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pte_m">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>51</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>各层质量(t)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>各层标高(m)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
